mongodb/spring-cloud-starter-stream-source-mongodb/src/test/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+collection=testing
+query={ }
+uri=mongodb://localhost:12345/test
```

Comment:
No `application.properties` in the `test` code base as well.
Everything must be utilized as `@IntegrationTest` or `@TestPropertySource`.
Sometime we add `System.setProperty`.
Anyway Embedded MongoDb must be with random port: http://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-nosql.html#boot-features-mongo-embedded